pyproject.toml

```diff
@@ -1,8 +1,10 @@
 [project]
 name = "charmcraft"
+dynamic = ["version"]
 description = "The main tool to build, upload, and develop in general the Juju charms."
-dynamic = ["version", "readme"]
+readme = "README.md"
 dependencies = [
+    "attrs<23",
```

Comment:
What drove this need? Please include a comment so we know when we can undo it

Reply:
The commit message says it all.

> Stick to older attr version to avoid hatchling UnknownPluginError (fancy-pypi-readme)